clientv3/concurrency/election.go

```diff
@@ -55,8 +55,24 @@ func ResumeElection(s *Session, pfx string, leaderKey string, leaderRev int64) *
 	}
 }
 
-// Campaign puts a value as eligible for the election. It blocks until
-// it is elected, an error occurs, or the context is cancelled.
+// Campaign puts a value as eligible for the election on the prefix
+// key.
+// Multiple Sessions can participate in the election for the
+// same prefix, but only one can be the leader at a time.
+// A Session succeeds in election if it can create a key with the
+// prefix before other participating Sessions. If there are other
+// keys created before with the same prefix, the session waits for
+// previous keys to be deleted by doing a Watch() for the Delete events
+// on the previous keys with the same prefix.
+//
+// If the context is 'context.TODO/Background', the Compaign will continue
```

Comment:
typo `Compaign` ...  @gyuho can verify the content at best, but couple of small comments here.